schema/vrs-source.yaml

```diff
@@ -613,90 +663,109 @@ definitions:
         type: string
         enum: [ "DefiniteRange" ]
         default: "DefiniteRange"
+        description: MUST be "DefiniteRange"
       min:
         type: number
+        description: The minimum value; inclusive
       max:
         type: number
+        description: The maximum value; inclusive
     required: ["type", "min", "max"]
 
+  IndefiniteRange:
+    description: >-
+      A half-bounded range of numbers represented as a number bound and
```

Comment:
While I agree that limiting this to "<=" and ">=" is correct for Integer values. It may not be sufficient for Real numbers. When Integers are the class of Numbers in the value field (or min and max fields for DefiniteRange) the equality portion of the comparator can be made on a precise Integer. But with Real numbers if one wanted to convey the concept of "less than 4" they wouldn't be able to express that precisely enough using the equality symbol (it could be 3.99999 or 3.99999999 or ...).
So I wonder if we jumped the gun on this or if we need to re-litigate this?
If we add back the ">" and "<" as options then we will introduce dual ways to convey the same Integer Ranges by either using the precise endpoint with the gt/lt equal symbol comparators or the prev/next endpoint with the gt/lt symbols. Ugh.

Reply:
Added #362 to address later.